src/tools/sssctl/sssctl_domains.c

```diff
@@ -66,6 +98,24 @@ errno_t sssctl_domain_list(struct sss_cmdline *cmdline,
         return EIO;
     }
 
+    if (verbose) {
+        for (i = 0; domains[i] != NULL; i++) {
+            ret = domain_is_subdomain_check(sifp, domains[i], &is_subdom);
+            if (ret != EOK) {
+                DEBUG(SSSDBG_CRIT_FAILURE, "Subdomain check failed\n");
+                return ret;
+            }
+
+            if (is_subdom == true) {
+                printf("Subdomain: %s\n", domains[i]);
+            } else if (is_subdom == false) {
```

Comment:
@justin-stephenson: Just a minor comment here ... I'd prefer something like:
```
if (is_subdom) {
    printf("Subdomain: %s\n", domains[i]);
} else {
    printf("Primary domain: %s\n", domains[i]);
}
```

The reasons for this are:
- There's no need to use else if as there are just two cases (and the first one is covered by the if)
- I try to use explicitly checks in all types but the boolean ones.